holonix/README.md

```diff
@@ -0,0 +1,128 @@
+# Holonix
+
+NixOS and `nix-shell` in other OS environments implements the HC ops tooling.
+
+## Scope
+
+Where `nix-shell` is supported Holonix is the standard approach for:
+
+- Installing dependencies
+- Running tests
+- Building wasm
+- Building binaries
+- Implementing CI scripts
+- Conductor management
+- Installing situational tooling (e.g. `cargo-edit` for versioning)
+- Release management, automation and deploying official binaries
+- BAU automation and reporting tasks**
+- Downloading prebuilt official binaries from github**
+- Managing and updating binaries across official releases**
+
+** Coming Soon!
```

Comment:
This seems a bit out of place, what's coming soon?

Reply:
@zippy the two things with `**` next to them!